### PR TITLE
Proj4 is no more, rename to 'Proj'

### PR DESCRIPTION
**Pull request by nyalldawson**
Changes references to "proj4" to just "proj". 4 was a version number, not a part of the name!

### CODE REVIEW

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst [hunk 1]
```diff
@@ -148,7 +148,7 @@ including:
 
 * the Layer name to display in the :guilabel:`Layers` panel
 * setting the Coordinate Reference System: Displays the layer’s Coordinate
-  Reference System (CRS) as a PROJ.4 string. You can change the layer’s CRS by
+  Reference System (CRS). You can change the layer’s CRS by
```
Comment (nyalldawson) [inline source/docs/user_manual/working_with_mesh/mesh_properties.rst]: What's the correct way to link this to https://github.com/qgis/QGIS-Documentation/blob/6c8f5d497f9209e7cf0f48d5a251436a1a813a23/source/docs/user_manual/working_with_projections/working_with_projections.rst#layer-coordinate-reference-systems ?

Comment (DelazJ) [reply source/docs/user_manual/working_with_mesh/mesh_properties.rst]: insert an anchor above the line you want to link and use :ref:\`my label \<my_anchor\>\` tag here to call it (see https://docs.qgis.org/testing/en/docs/documentation_guidelines/writing.html#label-reference)

Comment (nyalldawson) [reply source/docs/user_manual/working_with_mesh/mesh_properties.rst]: Thanks, done

**Comment by nyalldawson** [comment]
Yep, all done

**Comment by havatv** [comment]
Thanks, @nyalldawson!

**Pull request merged**